Dockerfile

```diff
@@ -19,11 +19,13 @@ RUN apt-get update && \
       git-core \
       gnupg2 \
       jq \
+      libasio-dev \
       libcaf-dev \
       libbroker-dev \
       libflatbuffers-dev \
       libfmt-dev \
       libpcap-dev tcpdump \
+      librestinio-dev \
```

Comment:
These need to be added to the instructions for building from source, and need to be installed in CI (where this plugin isn't being built yet).